crates/fluvio-version-manager/src/common/settings.rs

```diff
@@ -139,4 +163,26 @@ mod tests {
         remove_file(&settings_path).expect("Failed to remove settings.toml file");
         delete_fvm_dir();
     }
+
+    #[test]
+    fn creates_settings_file_if_not_exists_on_open() {
+        create_fvm_dir();
+
+        let settings_path =
+            Settings::settings_file_path().expect("Failed to get settings.toml path");
+
+        assert!(
+            !settings_path.exists(),
+            "the settings file should not exist at this point"
+        );
+
+        Settings::open().expect("Failed to create settings.toml file");
+
+        assert!(
+            settings_path.exists(),
+            "the settings file should exist at this point"
+        );
+
```

Comment:
Should add a readback and asserts after the create

Reply:
Sure thing! I should mention that `Settings` will be an empty file after creation, due to the fact that no version nor channel is set, just in case both assertions on `is_none` looks weird!